components/raftstore/src/store/fsm/apply.rs

```diff
@@ -56,8 +56,9 @@ use super::metrics::*;
 
 use super::super::RegionTask;
 
-const WRITE_BATCH_MAX_KEYS: usize = 128;
+const WRITE_BATCH_MAX_KEYS: usize = 256;
```

Comment:
Because `RocksWriteBatchVec` would be suitable for a larger batch. I think it will be better if we add an interface to trait WriteBatch and let itself judge whether it shall be written into `Engine`.